Services/Mail/classes/Mime/Subject/class.ilMailMimeSubjectBuilder.php

```diff
@@ -52,7 +50,7 @@ public function subject(string $subject, bool $addPrefix = false, string $contex
                 }
             }
 
-            if (strlen($prefix) > 0) {
+            if ($prefix !== '') {
```

Comment:
If we use this check, we have to ensure that the variable is of type `string`. It might be also `null` here.